program-runtime/src/compute_budget.rs

```diff
@@ -102,35 +102,53 @@ impl ComputeBudget {
         &mut self,
         message: &SanitizedMessage,
         requestable_heap_size: bool,
+        per_instruction_default: bool,
```

Comment:
heh, quite a nit

Reply:
`requestable_heap_size` is also named after the feature and I think it makes it easier to read and understand.
Otherwise one would have to look for the call site and figure out what was passed here.